python/pyspark/sql/functions/builtin.py

```diff
@@ -15534,19 +15532,7 @@ def to_csv(col: "ColumnOrName", options: Optional[Dict[str, str]] = None) -> Col
     |      2,Alice|
     +-------------+
 
-    Example 2: Converting a complex StructType to a CSV string
```

Comment:
Considering that `to_csv` no longer supports `complex type data`, we will remove this example.